src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/discount_list.html.twig

```diff
@@ -22,28 +22,30 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
-<table class="table discountList{% if discounts is empty %} d-none{% endif %}">
-  <thead>
-  <tr>
-    <th>{{ 'Name'|trans({}, 'Admin.Global') }}</th>
-    <th>{{ 'Value'|trans({}, 'Admin.Global') }}</th>
-    <th class="text-right d-print-none">{{ 'Actions'|trans({}, 'Admin.Global') }}</th>
-  </tr>
-  </thead>
-  <tbody>
-  {% for discount in discounts %}
+<div class="col-md-12">
+  <table class="table discountList{% if discounts is empty %} d-none{% endif %}">
```

Comment:
I am not sure if camel case in class names is the convention?

Reply:
Sorry, it is not newly introduced, it already exists like this in SCSS :facepalm:

Reply:
Yeah otherwise I agree with you ^^

Reply:
Probably introduced by me months ago, it's a miss :/